services/integration/src/controllers/project.js

```diff
@@ -0,0 +1,60 @@
+const Project = require('../model/projectModel');
+
+const { asyncHandler } = require('../utils');
+//routes
+const test = (req, res) => {
```

Comment:
why are you creating a function with this name `test`?

Reply:
It's already changed.